.github/workflows/base-lambdas-edge-deploy.yml

```diff
@@ -0,0 +1,99 @@
+name: 'Z-BASE Lambdas: action to deploy a single lambda'
+
+on:
+  workflow_call:
+    inputs:
+      environment:
+        required: true
+        type: string
+      python_version:
+        required: true
+        type: string
+      build_branch:
+        required: true
+        type: string
+        default: main
+      sandbox:
+        required: true
+        type: string
+        default: main
+      lambda_handler_name:
+        required: true
+        type: string
+      lambda_aws_name:
+        required: true
+        type: string
+    secrets:
+      AWS_ASSUME_ROLE:
+        required: true
+
+permissions:
+  pull-requests: write
+  id-token: write # This is required for requesting the JWT
+  contents: read # This is required for actions/checkout
+
+jobs:
+  python_deploy_lambda:
+    name: Deploy lambda ${{ inputs.lambda_handler_name }} to ${{ inputs.sandbox }}
+    runs-on: ubuntu-latest
+    environment: ${{ inputs.environment }}
+
+    steps:
+      - name: Display passed variables
+        run: |
+          echo Python Version: ${{ inputs.python_version }}
+          echo Build Branch: ${{ inputs.build_branch }}
+          echo Environment: ${{ inputs.environment }}
+          echo Sandbox: ${{ inputs.sandbox }}
+          echo Lambda Handler Name: ${{ inputs.lambda_handler_name }}
+          echo Lambda AWS function: ${{ inputs.lambda_aws_name }}
+
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          repository: 'nhsconnect/national-document-repository'
+          ref: ${{ inputs.build_branch }}
+          fetch-depth: '0'
+
+      - name: Set up Python ${{ inputs.python_version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ inputs.python_version }}
+
+      - name: Make virtual environment
+        run: make edge_env
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v2
+        with:
+          role-to-assume: ${{ secrets.AWS_ASSUME_ROLE }}
+          role-skip-session-tagging: true
+          aws-region: 'us-east-1'
```

Comment:
Can we store this in github vars?

Reply:
I don't think we'd need to as it's a hard coded value